polymetis/polymetis/python/scripts/launch_robot.py

```diff
@@ -79,15 +79,19 @@ def cleanup():
     signal.signal(signal.SIGTERM, lambda signal_number, stack_frame: cleanup())
 
     # Start client
-    t0 = time.time()
-    while not check_server_exists(cfg.ip, cfg.port):
-        time.sleep(0.1)
-        if time.time() - t0 > cfg.timeout:
-            raise ConnectionError("Robot client: Unable to locate server.")
-
-    log.info(f"Starting robot client...")
-    client = hydra.utils.instantiate(cfg.robot_client)
-    client.run()
+    if cfg.robot_client:
+        t0 = time.time()
+        while not check_server_exists(cfg.ip, cfg.port):
+            time.sleep(0.1)
+            if time.time() - t0 > cfg.timeout:
+                raise ConnectionError("Robot client: Unable to locate server.")
+
+        log.info(f"Starting robot client...")
+        client = hydra.utils.instantiate(cfg.robot_client)
+        client.run()
+
+    else:
+        signal.pause()
```

Comment:
Same question as the gripper.

Reply:
`launch_robot.py` was originally written to launch the server in a subprocess using `subprocess.run()` back when a client was always needed. I wanted to make minimal changes to existing files so I had the parent process hang, but will make it more elegant when the design of this PR is finalized.